README.md

```diff
@@ -74,6 +74,8 @@ manually if your distribution does not provide a package.
 
 First, make sure you have the required __dependencies__:
 - build:
+	- autopoint (for bootstrapping)
```

Comment:
autopoint is part of gettext, which is already listed. Or are there distros that ship this separately?